ddtrace/internal/writer.py

```diff
@@ -532,6 +533,10 @@ def write(self, spans=None):
                     # are initialized
                     if asbool(os.getenv("DD_INSTRUMENTATION_TELEMETRY_ENABLED", True)):
                         telemetry_writer.enable()
+
+                    if asbool(os.getenv("DD_REMOTE_CONFIGURATION_ENABLED", True)):
+                        RemoteConfig.enable()
```

Comment:
Can you help me understand how the `AgentWriter` takes care of the call to restart on fork now?